clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/ActiveActiveStoreIngestionTask.java

```diff
@@ -114,7 +115,9 @@ public ActiveActiveStoreIngestionTask(
       int errorPartitionId,
       boolean isIsolatedIngestion,
       Optional<ObjectCacheBackend> cacheBackend,
-      DaVinciRecordTransformerFunctionalInterface recordTransformerFunction) {
+      DaVinciRecordTransformerFunctionalInterface recordTransformerFunction,
+      Lazy<ZKHelixAdmin> zkHelixAdmin,
+      int port) {
```

Comment:
`VeniceStoreVersionConfig` can be used to extract `listenerPort` if necessary, so we don't need to explicitly pass this param.

Reply:
Sure